core/src/main/java/org/elasticsearch/index/fielddata/ScriptDocValues.java

```diff
@@ -253,124 +254,41 @@ public int size() {
             return values.count();
         }
 
-        public double factorDistance(double lat, double lon) {
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT);
-        }
-
-        public double factorDistanceWithDefault(double lat, double lon, double defaultValue) {
-            if (isEmpty()) {
-                return defaultValue;
-            }
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT);
-        }
-
-        public double factorDistance02(double lat, double lon) {
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT) + 1;
-        }
-
-        public double factorDistance13(double lat, double lon) {
-            GeoPoint point = getValue();
-            return GeoDistance.FACTOR.calculate(point.lat(), point.lon(), lat, lon, DistanceUnit.DEFAULT) + 2;
-        }
-
         public double arcDistance(double lat, double lon) {
```

Comment:
My motivation is both making it so there is one obvious way to calculate distance (I was reminded recently of this beatiful mantra from the Zen of Python: `There should be one— and preferably only one —obvious way to do it.`).  I also think not having instance methods will allow us to play more with the underlying field access so we dont need an intermediate object, GeoPoint).